include/alpaka/dev/DevGenericSycl.hpp

```diff
@@ -116,7 +116,7 @@ namespace alpaka::experimental
     {
     public:
         DevGenericSycl(sycl::device device, sycl::context context)
-            : m_impl{std::make_shared<detail::DevGenericSyclImpl>(std::move(device), std::move(context))}
+            : m_impl{std::make_shared<trait::detail::DevGenericSyclImpl>(std::move(device), std::move(context))}
```

Comment:
Please revert.

Reply:
I have just reverted all of the `detail namespace` changes. Now everything is back into `alpaka::detail`